Code/Editor/Core/QtEditorApplication.cpp

```diff
@@ -415,33 +415,37 @@ namespace Editor
         }
 
         // Ensure that the Windows WM_INPUT messages get passed through to the AzFramework input system.
-        // These events are now consumed both in and out of game mode.
-        if (msg->message == WM_INPUT)
+        // These events are only broadcast in game mode. In Editor mode, RenderViewportWidget creates synthetic
+        // keyboard and mouse events via Qt.
+        if (GetIEditor()->IsInGameMode())
```

Comment:
Does this disable the ability to trigger ImGui while not in game mode at all?

Reply:
IMGUI outside of game mode still works, you just need to have viewport focus (see the gif above for a demo).

We still get keyboard events from the viewport, just not the "raw" windows events.

Reply:
Ok that's what I figured from the description, just needed to double check. Looks good to me, thanks!